atomix/cluster/src/main/java/io/atomix/raft/impl/RaftContext.java

```diff
@@ -186,7 +184,6 @@ public RaftContext(
 
     // Construct the core log, reader, writer, and compactor.
     raftLog = storage.openLog();
-    logReader = raftLog.openUncommittedReader();
```

Comment:
The reader is used only rarely. Only once in case of the Leader, and only in rare cases if it is a follower. So keeping a reader opened in the follower will result in the same bug. Whenever the reader is used in the follower, it seeks to a specific position. Seek operation is more expensive than creating a new reader. So we donot gain anything by keeping a reader opened.

Reply:
Thanks :+1: